src/core/config/tabular_data/input_tableset_type.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <memory>
+
+#include "model/table/idataset_stream.h"
+
+namespace config {
+
+struct InputTableSet {
+    std::string name;
+    std::vector<std::shared_ptr<model::IDatasetStream>> parsers;
+};
```

Comment:
I believe this type should be defined like this:
```suggestion
using InputTableSet = std::vector<InputTable>;
```

Reply:
nit: But I would think about using the word `set` in the naming --- this may lead to associations with the fact that we store tables using `std::set`.
I would like to know the opinion of reviewers who will also review this pull request.

Reply:
You can use `IDatasetStream::GetRelationName()` - it seems that this will be enough.
Also now the `name` field looks a little strange, in one case it is the name of the directory (when we pass the path to the directory with files), in the other case it is the file itself (when the path to the file is passed).

Suggested name `InputTableCollection` acceptable

Reply:
`name` here is not the name of a table, but the name of a _whole_ dataset (a collection of tables). In the current approach the user specifies folder name, where csv-s are locaded, and this folder name becomes `InputTableSet.name`.

Reply:
In a personal dialogue we discussed it in more detail and decided that we would remove the `name` field